exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -66,21 +67,12 @@ func (b *Bitfinex) WsSend(data interface{}) error {
 	if err != nil {
 		return err
 	}
-	return b.WebsocketConn.WriteMessage(websocket.TextMessage, json)
-}
-
-// WsSubscribe subscribes to the websocket channel
-func (b *Bitfinex) WsSubscribe(channel string, params map[string]string) error {
-	req := make(map[string]string)
-	req["event"] = "subscribe"
-	req["channel"] = channel
-
-	if len(params) > 0 {
-		for k, v := range params {
-			req[k] = v
-		}
+	if b.Verbose {
+		log.Debugf("Sending message to websocket %v", data)
```

Comment:
Please include the exchange name here for verbose output otherwise it will be hard to keep track of which exchange is sending what data when multiple exchanges are enabled